tfx/experimental/pipeline_testing/examples/chicago_taxi_pipeline/taxi_pipeline_regression_e2e_test.py

```diff
@@ -73,7 +92,7 @@ def testTaxiPipelineBeam(self):
         pipeline_name=self._pipeline_name,
         run_id=None)
 
-    # Run pipeline with stub executors first time.
+    # Run pipeline with stub executors.
     taxi_pipeline = taxi_pipeline_beam._create_pipeline(  # pylint:disable=protected-access, unexpected-keyword-arg
```

Comment:
Out of curiosity, why do we need 'unexpected-keyword-arg' here?
Could you match the order of arguments with the function definition?

Reply:
Could you answer above question? or `unexpected-keyword-arg` is not need here, either?